presidio-analyzer/tests/test_transformers_recognizer.py

```diff
@@ -23,13 +23,6 @@ def nlp_engine(nlp_engines):
     return nlp_engine
 
 
-def prepare_and_analyze(nlp, recognizer, text, entities):
```

Comment:
Does removing this have an impact on memory usage?

Reply:
Sorry for delay in responding @omri374, I missed your comment.

The specific impact on memory usage is the redundant `nlp.load()` call in the `prepare_and_analyze` function. `load()` is already called in the `nlp_engine` fixture.

Happy to revert back to using the `prepare_and_analyze` function, minus the load call if you'd prefer

Reply:
That's be great. Thanks.

Reply:
Done